gammapy/irf/tests/test_hess_chains.py

```diff
@@ -33,9 +34,43 @@ def get_list_of_chains():
 @requires_dependency('scipy')
 @requires_dependency('yaml')
 @requires_data('gammapy-extra')
-def test_EffectiveArea(data_manager, chain):
-    ref_file = make_path(chain['aeff2D_reference_file'])
-    ref_aeff = open(str(ref_file), 'r').read()
+def test_hess_chains(data_manager, chain):
+
+    ref_file_aeff = make_path(chain['aeff2D_reference_file'])
+    ref_aeff = open(str(ref_file_aeff), 'r').read()
+    ref_file_edisp = make_path(chain['edisp2D_reference_file'])
+    ref_edisp = open(str(ref_file_edisp), 'r').read()
+    ref_file_psf = make_path(chain['psf_reference_file'])
+    ref_psf = open(str(ref_file_psf), 'r').read()
+    ref_file_obs = make_path(chain['obs_reference_file'])
+    ref_obs = open(str(ref_file_obs), 'r').read()
+    ref_file_loc = make_path(chain['location_reference_file'])
+    ref_loc = open(str(ref_file_loc), 'r').read()
+
+    obs_nr = chain['obs']
+    obs_id = chain['obs_id']
+
     store = data_manager[chain['store']]
-    aeff = store.obs(obs_id=chain['obs']).aeff
-    assert aeff.info() == ref_aeff
+    obs = store.obs(obs_id=obs_id)
+
+    assert str(obs.location(hdu_type='events').path(abs_path=False)) == ref_loc
+
+    # Todo : Find a better way to capture stdout
+    # see http://stackoverflow.com/questions/5136611/capture-stdout-from-a-script-in-python/10743550#10743550
+    import sys
+    from StringIO import StringIO
```

Comment:
This doesn't work in Python 3:
```python
$ python3 -c 'import StringIO'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named 'StringIO'
```
Also: this should really be done in a with statement, which guarantees that `sys.stdout` is restored when it exists.

Bottom line: you have to put this in `gammapy.utils` and add a test that captures something that's printed to show that it works on Python 2 and 3.
Let me know if you want me to add this.